### DISCUSSION TITLE
Use TrackIR 5 Camera with opentrack

### DISCUSSION THREAD

**Discussion by teeter477** [opentrack/opentrack#1422] [Q&A]
Is it possible to use the TrackIR camera with opentrack? I ask because I'm looking for a calibration solution to compensate for the fact that my TrackIR is mounted high (on top of a TV), looking down at perhaps a ~30 degree angle, in order to see my TrackClip Pro. I don't have issues with the camera seeing the LEDs, however the angle affects the tracking as it's not straight-on as the TrackIR software expects.

My understanding is that opentrack has calibration settings to compensate for this, although I can't find a way to enable the TrackIR itself as an Input for opentrack to use.

Just thinking out loud (and I may be completely full of it) - if there's no way to directly use the TrackIR camera within opentrack, could there perhaps be a way to have the TrackIR software running, "capture" the output of the TrackIR software with opentrack as an input, and then use opentrack to apply the translation and output to the game?

**Comment by sthalik** [comment]
opentrack will never support NaturalPoint hardware due to abusive legal action taken by the company against the Freetrack project. Supporting their hardware will incentivize people into buying it in the first place. Making these bastards earn money off of opentrack isn't acceptable.

You may be interested in the [linuxtrack](https://github.com/uglyDwarf/linuxtrack) project.

**Comment by kitkat0981** [comment]
if i would have known earlier, i woulnd not have bought Trackir...
ok, so which hardware do you recommend? I'm not necessarely looking for the cheapest since i did spend 350$cad on this TrackIR Pro.

thanks for responding

**Reply by sthalik** [reply]
Get a used PS3 Eye -- one with the IR filter already removed, and the FOV slider still working. If it comes with a visible light filter already in the case it's even better.

**Reply by pivitron** [reply]
try Beam Eye Tracker! WOrks like a charm and supports head and eye tracking like tobii eye without lag https://beam.eyeware.tech/?via=pivi

**Comment by merlinfive** [comment]
Delanclip…, works perfect for me

**Comment by photovirus** [comment]
Turned out [it is possible.](https://forum.il2sturmovik.ru/topic/12924-trackir5-glovepie-python3-opentrack-расширенное-управление-обзором/)

Author set up GlovePIE and some python scripts to forward TrackIR data to opentrack via UDP. Haven't tried it yet, but I definitely intend to.

**Reply by NorthernPryde** [reply]
I know this is a year out what I bought a trackir five need to know how to get it to work in opentrack. 
Please message me your email me 

jdevos19@gmail.com

**Reply by photovirus** [reply]
In the end, I forfeited the idea as too inconvenient. So I’m not sure I can help, but I’ll try to answer your questions anyway.

**Reply by NorthernPryde** [reply]
big thanks on the reply.
I am interested in getting this Trackir to work in Opentracker. 
I play Project Reality the game is old but has a huge player base. 
We are able to get wedcam to work as motion input for the game in Opentracker.
I bought the Trackir thinking it would be best for the this. Your post is the only one i could find on info to do this.
any help would be great. I plan on trying the video. it is in Russian so I'll do my best.

**Reply by photovirus** [reply]
I’m not sure if the video has all the info. 

Try auto translating the actual forum page. Should work decently. 

And head over [to this new forum thread](https://forum.il2sturmovik.ru/topic/15418-trackir-freepie-vjoy-opentrack-aom_upgrade/) with a (hopefully working) link to files.

**Comment by NorthernPryde** [comment]
I'm new to the track ir5 and could use some assistance in trying to get it to work with opentrack. I know it's not supported but everybody has a workaround.
